src/sensors/gps_display.cpp

```diff
@@ -291,11 +298,11 @@ void GPSDisplay::debugPositionUpdate()
 		return;
 	
 	QTime now = QTime::currentTime();
-	float offset = now.msecsSinceStartOfDay() / (float)(10 * 1000);
-	float accuracy = 12 + 7 * qSin(2 + offset);
-	float altitude = 400 + 10 * qSin(1 + 0.1f * offset);
+	const auto offset = now.msecsSinceStartOfDay() / qreal(10 * 1000);
+	const auto accuracy = float(12 + 7 * qSin(2 + offset));
+	const auto altitude = 400 + 10 * qSin(1 + offset / 10);
 	
-	MapCoordF coord(30 * qSin(0.5f * offset), 30 * qCos(0.53f * offset));
+	MapCoordF coord(30 * qSin(0.5 * offset), 30 * qCos(0.53 * offset));
```

Comment:
Then it would probably turn this into `const auto coord = MapCoordF{...};`. This would make reviews even more difficult. And this particular line is debugging code which will need a refactoring anyway.

Reply:
ES.23 only asks to "Prefer the {} initializer syntax", and it points out that you must be careful with initializer lists. But it doesn't say you should prefer 
~~~
 const auto now { QTime::currentTime() };
~~~
over
~~~
 const auto now = QTime::currentTime();
~~~
And only the latter seems to allow maximum consistency, aka almost-always-auto (AAA) style, as in:
~~~
        const auto now = QTime::currentTime();
        const auto offset = now.msecsSinceStartOfDay() / qreal(10 * 1000);
        const auto accuracy = float(12 + 7 * qSin(2 + offset));
        const auto altitude = 400 + 10 * qSin(1 + offset / 10);

        const auto coord = MapCoordF{ 30 * qSin(0.5 * offset), 30 * qCos(0.53 * offset) };
~~~
(I left C/operator-style conversions for `qreal` and `float`. And I didn't go so far to use "east-side const".)
This recent article and comments are related: https://www.fluentcpp.com/2018/09/28/auto-stick-changing-style/

By the way, Qt Creator's formatting stumples over
~~~
const auto foreground = QColor{tracking_lost ? Qt::gray : Qt::red};
~~~
i.e. ternary operator in `{}? initializer. That's why I went back to `()` in this spot.